packages/ckeditor5-indent/docs/features/indent.md

```diff
@@ -5,7 +5,7 @@ category: features
 
 {@snippet features/build-indent-source}
 
-The block indentation feature allows you to set indentation for text blocks such as paragraphs or headings and lists. It is implemented by three plugins: {@link module:indent/indent~Indent}, {@link module:indent/indentblock~IndentBlock} and {@link module:list/list~List}.
+The block indentation feature allows you to set indentation for text blocks such as paragraphs or headings and lists. It is implemented by three plugins: {@link module:indent/indent~Indent}, {@link module:indent/indentblock~IndentBlock} and {@link module:list/list~List}. Its main purpose is to visually distinguish parts of the content. It maybe structure elements like headings or lists, or parts of text like paragraphs that need to be emphasized, present quotes or need to be otherwise separated from the main content body.
```

Comment:
```suggestion
The block indentation feature allows you to set indentation for text blocks such as paragraphs or headings and lists. It is implemented by three plugins: {@link module:indent/indent~Indent}, {@link module:indent/indentblock~IndentBlock} and {@link module:list/list~List}. Its main purpose is to visually distinguish parts of the content. It may be structure elements like headings or lists, or parts of text like paragraphs that need to be emphasized, present quotes or need to be otherwise separated from the main content body.
```